### PR TITLE
Add BMP3xx and BMA423 drivers

### PR DESCRIPTION
**Pull request by mattytrentini**
Add an accelerometer and pressure sensor driver. 

Both were listed in this MicroPython discussion: [Implementing an IMU/accelerometer/gyro driver? Use the FIFO!](https://github.com/orgs/micropython/discussions/15512)

I've marked both drivers with "⏩" to indicate that the driver supports FIFO access. That was [suggested in that discussion](https://github.com/orgs/micropython/discussions/15512#discussioncomment-10106077) by @Josverl and I think it's a good idea to try out. Would be good to review other drivers and add the same badge where appropriate.

### CODE REVIEW

**Comment by mattytrentini** [comment]
(I also took the opportunity to fix the broken link to the _Getting Started..._ book. The second version was released and the original link appears to have been removed.)

File: readme.md [hunk 1]
```diff
@@ -753,6 +753,7 @@ Other places you can look for MicroPython Libraries:
 * [MicroPython_ADXL343](https://github.com/jposada202020/MicroPython_ADXL343) - MicroPython Driver for the Analog Devices ADXL343 Accelerometer.
 * [MicroPython_BMA220](https://github.com/jposada202020/MicroPython_BMA220) - MicroPython Driver for the Bosch BMA220 Accelerometer.
 * [MicroPython_BMA400](https://github.com/jposada202020/MicroPython_BMA400) - MicroPython Driver for the Bosch BMA400 Accelerometer.
+* [bma423-pure-mp](https://github.com/antirez/bma423-pure-mp) - MicroPython Drive for the Bosch 423 accelerometer. Includes FIFO support. ⏩
```
Comment (andypiper) [inline readme.md:756]: nitpick -> Driver not Drive

**Review by andypiper — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (mattytrentini) [reply readme.md]: Thanks @andypiper! Fixed.

**Review by mcauser — APPROVED** [review]

**Comment by mcauser** [comment]
Thanks guys!

**Comment by mcauser** [comment]
Maybe we should add an emoji for mip installable 🤔

**Comment by Josverl** [comment]
It's always hard to see how the icons look across platforms 
But the suitcase/package on the conveyor perhaps ?
🛄

**Pull request merged**